ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/DC1-BL1A.cfg

```diff
@@ -417,6 +428,10 @@ router bgp 65104
       route-target export evpn 65104:31
       router-id 192.168.255.14
       redistribute connected
+      !
+      address-family ipv4
+      !
+      address-family ipv6
```

Comment:
Please look into this. You should not be creating these address families unless there is something configured under them. Thanks.

Reply:
Done!